File: solutions/Python/darts/2/darts.py
def score(x, y):
    if ((x * x ) + (y * y)) ** (1 / 2) <= 1:
        return 10
    if ((x * x ) + (y * y)) ** (1 / 2) <= 5 :
        return 5
    if ((x * x ) + (y * y)) ** (1 / 2) <= 10:
         return 1 
    return 0
   
    
     

File: solutions/C/darts/2/darts.c
#include "darts.h"
#include <math.h>

uint8_t score(coordinate_t coordinate) {
    float radiusSquared = coordinate.x * coordinate.x + coordinate.y * coordinate.y;
    if (radiusSquared <= 1.0f) {
        return 10;
    }
    if (radiusSquared <= 25.0f) {
        return 5;
    }
    if (radiusSquared <= 100.0f) {
        return 1;
    }
    return 0;
}

